src/main/java/org/opensearch/knn/index/codec/KNN920Codec/KNN920Codec.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.knn.index.codec.KNN920Codec;
+
+import org.apache.lucene.codecs.Codec;
+import org.apache.lucene.codecs.CompoundFormat;
+import org.apache.lucene.codecs.DocValuesFormat;
+import org.apache.lucene.codecs.FilterCodec;
+import org.opensearch.knn.index.codec.KNNFormatFacade;
+import org.opensearch.knn.index.codec.KNNFormatFactory;
+
+import static org.opensearch.knn.index.codec.KNNCodecFactory.CodecDelegateFactory.createKNN92DefaultDelegate;
+
+/**
+ * KNN codec that is based on Lucene92 codec
+ */
+public final class KNN920Codec extends FilterCodec {
+
+    private static final String KNN920 = "KNN920Codec";
+    private final KNNFormatFacade knnFormatFacade;
+
+    /**
+     * No arg constructor that uses Lucene91 as the delegate
+     */
+    public KNN920Codec() {
+        this(createKNN92DefaultDelegate());
+    }
+
+    /**
+     * Constructor that takes a Codec delegate to delegate all methods this code does not implement to.
+     *
+     * @param delegate codec that will perform all operations this codec does not override
+     */
+    public KNN920Codec(Codec delegate) {
+        super(KNN920, delegate);
+        knnFormatFacade = KNNFormatFactory.createKNN910Format(delegate);
```

Comment:
Why create 910 here?

Reply:
+1

Reply:
my bad, let me add 920 format and update this code